tf.go

```diff
@@ -17,7 +16,7 @@ import (
 type TF struct {
 	ctx         context.Context
 	name        string
-	writer      io.Writer
+	output      Output
```

Comment:
> // Standard output is for information that is expected from the executed tasks.

Shouldn't we just to pass the Standard writer here? Then we would use the Messaging just in `flowRunner` when bad usage is reported. The Messaging writer could still be handy in future e.g. if we allow using custom a flow runner.